index.js

```diff
@@ -1,18 +1,47 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "skyblue";
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const travName = document.querySelector("#profile h2")
+travName.textContent = traveler.name
+
+const travNickName = document.querySelector("#profile em") 
+travNickName.textContent = traveler.nickname
 
+const travPhoto = document.querySelector("#profile img")
+travPhoto.src = traveler.photo
+travPhoto.alt = traveler.name 
 
 /***** Deliverable 4 *****/
+list = document.querySelector("#animals")
+
+traveler.animalSightings.forEach(function(obj){
```

Comment:
This is good. It would be better if you had a more descriptive name for the parameter, however. Instead of `obj`, maybe `animalObj`or `sightingObj`